src/shared/utils/utils.ts

```diff
@@ -8,8 +8,8 @@ export function removeProperty<
   return objWithoutProperty;
 }
 
-export const formatLargeNumber = (x: number) =>
-  x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+export const formatLargeNumber = (x?: number | null) =>
```

Comment:
This is not exposed as a library export.
I agree, we should use one instance, but that's out of the scope of this PR.
We'd need to go over the utility functions, hooks, and other things that can be reused and put them in a library. Franklin is supposed to be a component library, so case can be made to either extend it to more than components, or to have another library for utils and others.

Reply:
LongNumber is exposed from franklin as a component and just wraps this fn:
```
const LongNumber: FC<Props> = ({ children }: Props) =>
  formatLargeNumber(children)
```
but I guess this will mess up the types.

Reply:
🤔 Hmm I see, I've check where this is used and will try to remove the few places where it's used and replace with LongNumber then, if this affects types I'll abort for now

Reply:
OK, done now